administrator/components/com_admin/src/Model/SysinfoModel.php

```diff
@@ -287,6 +287,12 @@ public function &getConfig(): array
             $this->config[$key] = 'xxxxxx';
         }
 
+        foreach ($this->config as $key => $value) {
+			if (\is_array($value)) {
+				$this->config[$key] = \json_encode($value);
+			}
+		}
```

Comment:
```suggestion
            if (\is_array($value)) {
                $this->config[$key] = \json_encode($value);
            }
        }
```
Fix code style: On the 4.2-dev branch we have recently changed to use the PSR-12 standard. This uses spaces and not tabs for indentation in PHP files. Each indentation level is 4 tabs.